### PR TITLE
Update Client Library and API Integration for Backend Changes

### PR DESCRIPTION
**Pull request by vividroyjeong**
- Update client library and API integration to match backend changes
- Re-config proxy for base path(start with '/api') change in backend api

### CODE REVIEW

File: frontend/src/services/vdyp-api/apis/run-dcsvprojection-api.ts [hunk 1]
```diff
@@ -0,0 +1,152 @@
+/* tslint:disable */
```
Comment (dhlevi) [inline frontend/src/services/vdyp-api/apis/run-dcsvprojection-api.ts:1]: We shouldn't be disabling tslint. If you're getting linting errors, it's better to correct them vs. disabling the checks.

Note that Sonar will check them anyway and they'll need to be corrected regardless.

File: frontend/src/services/vdyp-api/apis/run-dcsvprojection-api.ts [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+// @ts-ignore
+import { BASE_PATH, BaseAPI, RequiredError } from '../base'
+import type { RequestArgs } from '../base'
+import type { FileUpload, Parameters } from '../models'
+
+export const RunDCSVProjectionApiAxiosParamCreator = function (
+  configuration?: Configuration,
+) {
+  return {
+    projectionDcsvPostForm: async (
```
Comment (dhlevi) [inline frontend/src/services/vdyp-api/apis/run-dcsvprojection-api.ts:16]: Why is this async? It includes a promise I'm just not sure why?

Comment (dhlevi) [reply frontend/src/services/vdyp-api/apis/run-dcsvprojection-api.ts:16]: I see the axios config, I just don't see it getting executed

Comment (dhlevi) [reply frontend/src/services/vdyp-api/apis/run-dcsvprojection-api.ts:16]: Is it only for the factory function below?

File: frontend/src/services/vdyp-api/apis/run-hcsvprojection-api.ts [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+import globalAxios from 'axios'
+import type { AxiosResponse, AxiosInstance, AxiosRequestConfig } from 'axios'
+import { Configuration } from '../configuration'
+// Some imports not used depending on template conditions
+// @ts-ignore
+import { BASE_PATH, BaseAPI, RequiredError } from '../base'
+import type { RequestArgs } from '../base'
+import type { FileUpload, Parameters } from '../models'
+
+export const RunHCSVProjectionApiAxiosParamCreator = function (
```
Comment (dhlevi) [inline frontend/src/services/vdyp-api/apis/run-hcsvprojection-api.ts:12]: Same as previous comment, should this be a promise? Doesn't seem to execute anything in an async way

File: frontend/src/services/vdyp-api/apis/run-hcsvprojection-api.ts [hunk 4]
```diff
@@ -0,0 +1,170 @@
+/* tslint:disable */
```
Comment (dhlevi) [inline frontend/src/services/vdyp-api/apis/run-hcsvprojection-api.ts:1]: As above, do not disable tslint/eslint

File: frontend/src/services/vdyp-api/apis/run-scsvprojection-api.ts [hunk 5]
```diff
@@ -0,0 +1,302 @@
+/* tslint:disable */
```
Comment (dhlevi) [inline frontend/src/services/vdyp-api/apis/run-scsvprojection-api.ts:1]: As above, do not disable tslint/eslint

File: frontend/src/services/vdyp-api/apis/run-scsvprojection-api.ts [hunk 6]
```diff
@@ -0,0 +1,302 @@
...
+      let baseOptions
+      if (configuration) {
+        baseOptions = configuration.baseOptions
+      }
+      const localVarRequestOptions: AxiosRequestConfig = {
+        method: 'POST',
+        ...baseOptions,
+        ...options,
+      }
+      const localVarHeaderParameter = {} as any
```
Comment (dhlevi) [inline frontend/src/services/vdyp-api/apis/run-scsvprojection-api.ts:56]: Ideally we don't use `any`. We should define a type for this if possible.

**Review by dhlevi — CHANGES_REQUESTED** [review]
We shouldn't be disabling ts/es linting. If you're seeing errors popping up, try to correct them appropriately vs. disabling the rules.

Couple questions on the parameter factories around the use of async/promises. If they're required, no worries, just wasn't sure what the purpose of them was.

**Comment by vividroyjeong** [comment]
@dhlevi Thank you for your last review. I've made the following changes to address your concerns. Please review again.
- Remove tslint/eslint disabling
- Remove unused dcsvprojection/scsvprojection api client libraries 
  : Remove csvprojection/scsvprojection api client libraries, which was causing issues with using async/promises. Currently it is not 
    being used, it is only declared in the API, not implemented. The client library was generated incorrectly. I've removed it to avoid 
    confusion. If it is implemented in the API in the future and we need it, I'll create it accordingly.
- Clean up - Remove unnecessary comments and unused import class

**Review by dhlevi — APPROVED** [review]

**Pull request merged**